tests/generators/index.html

```diff
@@ -4,20 +4,9 @@
 <meta charset="utf-8">
 <title>Blockly Generator Tests</title>
 <script>
-  var BLOCKLY_BOOTSTRAP_OPTIONS = {
-    scripts: [
-      'build/msg/en.js',
-      'tests/generators/unittest_javascript.js',
-      'tests/generators/unittest_python.js',
-      'tests/generators/unittest_php.js',
-      'tests/generators/unittest_lua.js',
-      'tests/generators/unittest_dart.js',
-      'tests/generators/unittest.js',
-    ],
-  }
-</script>
-<script src="../bootstrap.js"></script>
-<script>
+// N.B.: This script depends on the following (module) script to load
```

Comment:
Nit: can you swap the order of the scripts so that the later one depends on the earlier one? It'll still need a comment about the dependency, but at least the ordering will be intuitive.

Reply:
Counterpoint: they are in the order they actually get executed in.  (As a rough approximation: all ordinary `<script>`s get executed before any `<script type="module">`s.)

Does that change your opinion about what order they should be in?

In addition to reordering, I could rewrite the main, plain `<script>` as a module script, but that is somewhat more work.

Reply:
Counterpoint is reasonable. Fine to leave as is.